mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -726,3 +760,48 @@ func testAccMongoDBAtlasClusterAWSConfigdWithLabels(projectID, name, backupEnabl
 		}
 	`, projectID, name, backupEnabled, tier, region, labelsConf)
 }
+
+func testAccMongoDBAtlasClusterConfigWithPrivateEndpointLink(awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID, clusterName string) string {
+	return fmt.Sprintf(`
+		provider "aws" {
+			region     = "%[5]s"
+			access_key = "%[1]s"
+			secret_key = "%[2]s"
+		}
```

Comment:
I got an error when I tested it `Not a valid region: US_EAST_1` this is because the region format for AWS resource must to be as a following: `us-east-1`

Reply:
you can use the following function to resolve it: `${lower(replace("%s", "_", "-"))} `

Reply:
oh, I see, let me fix that.